packages/drivers/routerlicious-driver/src/wholeSummaryDocumentStorageService.ts

```diff
@@ -233,4 +253,24 @@ export class WholeSummaryDocumentStorageService implements IDocumentStorageServi
         });
         await Promise.all(blobCachePutPs);
     }
+
+    private convertSnapshotAndBlobsToSummaryTree(
+        snapshot: ISnapshotTree,
+        blobs: Map<string, ArrayBuffer>,
+    ): ISummaryTree {
+        const builder = new SummaryTreeAssembler();
```

Comment:
nit: This should be called `assembler` to keep it consistent with SummaryTreeAssembler.